templates/flavors/aad/machine-deployment.yaml

```diff
@@ -45,6 +45,7 @@ spec:
   template:
     spec:
       useExperimentalRetryJoin: true
```

Comment:
Thanks for this submission!

What we actually need to do is to remove all instances of the line `useExperimentalRetryJoin: true` entirely. (And we'll need to remove the addition of the deprecation message.)

Once we've removed all of the `useExperimentalRetryJoin: true` lines from template reference files, run `make generate-flavors` to ensure that all of the rendered templates are updated.

Reply:
Got it! Will do